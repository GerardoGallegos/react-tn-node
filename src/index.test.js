/* globals test, expect */

import React from 'react'
import { configure } from 'enzyme'
import EnzymeAdapter from 'enzyme-adapter-react-16'
import renderer from 'react-test-renderer'
import Note from './index'
import 'jest-styled-components'

configure({ adapter: new EnzymeAdapter() })

test('Should render the note component', () => {
  const note = renderer.create(<Note />).toJSON()
  expect(note.type).toEqual('div')
  expect(note).toMatchSnapshot()
})
