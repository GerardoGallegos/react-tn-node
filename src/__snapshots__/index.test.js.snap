// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the note component 1`] = `
.c2 {
  width: 20px;
  height: 20px;
  border-radius: 50%;
  background-color: red;
  background: url(https://ui-avatars.com/api/?background=FF0000&color=fff&size=200&name=J+G&rounded=true);
  background-size: cover;
}

.c0 {
  width: 100%;
  height: 100%;
  padding: .5em;
  cursor: pointer;
  -webkit-transition: .2s ease-out;
  transition: .2s ease-out;
  border-radius: 3px;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  box-sizing: border-box;
  position: relative;
}

.c0:hover {
  background: #e9f3f5;
}

.c6 {
  font-size: 13px;
  color: #c5c5c5;
  font-weight: 100;
}

.c4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-justify;
  -webkit-justify-content: space-space-between;
  -ms-flex-pack: space-justify;
  justify-content: space-space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  padding-top: .5em;
}

.c5 {
  font-size: 10px;
  margin: 0 .5em;
}

.c3 {
  max-width: 90%;
  height: auto;
  font-size: 15px;
  margin: 0 1em;
  text-align: left;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-justify;
  -webkit-justify-content: space-space-between;
  -ms-flex-pack: space-justify;
  justify-content: space-space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<div
  className="c0"
  onDoubleClick={[Function]}
>
  <div
    className="c1"
  >
    <div
      className="c2"
      size="20px"
      src="https://ui-avatars.com/api/?background=FF0000&color=fff&size=200&name=J+G&rounded=true"
    />
    <div
      className="c3"
    >
      
    </div>
  </div>
  <div
    className="c4"
  >
    <span
      className="c5"
    >
      
    </span>
    <span
      className="c6"
    >
      
    </span>
  </div>
</div>
`;
